ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/Model.hs

```diff
@@ -537,35 +537,35 @@ iteratorClose :: IteratorId -> Model blk -> Model blk
 iteratorClose itrId m = m { iterators = Map.insert itrId [] (iterators m) }
 
 {-------------------------------------------------------------------------------
-  Readers
+  Followers
 -------------------------------------------------------------------------------}
 
-readerExists :: CPS.ReaderId -> Model blk -> Bool
-readerExists rdrId = CPS.readerExists rdrId . cps
+followerExists :: CPS.FollowerId -> Model blk -> Bool
+followerExists rdrId = CPS.followerExists rdrId . cps
 
-checkIfReaderExists :: CPS.ReaderId -> Model blk
+checkIfFollowerExists :: CPS.FollowerId -> Model blk
```

Comment:
Just sharing an observation. Alignment breaks like this one are the reason we chose to not align `->` with `::` in the `ouroboros-consensus/doc/StyleGuide.md`. (Note that we also chose to keep `::` on the first line for other reasons.)

So, as your judgement call, if you feel like you're fixing up this kind of alignment for almost all of the signatures in some module, feel free to also change all those signatures in such a module to adhere to the `StyleGuide.md`. EG in this case (depending how many of the other signatures in this file you're already touching), it'd change to:

```
checkIfFollowerExists ::
     CPS.FollowerId
  -> Model blk
  -> a
  -> Either (ChainDbError blk) a
```

Somewhat separately: in the case of a rename, I would be tempted to have separate commits for the rename and for the alignment-fixup. Not crucial, but I think that would make it easier for me personally to review, for example.